setup.py

```diff
@@ -19,7 +19,7 @@
     ],
     install_requires=[
         'boto3>=1.3,<1.4',
-        'marshmallow>=2.7.0,<2.8.0',
+        'schematics',
```

Comment:
Fixed, as we no longer pin a version in this library - we now test with the recommended `--pre` version of `schematics`.